src/TodoProvider.js

```diff
@@ -0,0 +1,37 @@
+import React, { createContext, useState, useContext } from "react";
+import { v4 } from "uuid";
+
+const TodoContext = createContext();
+export const useTodo = () => useContext(TodoContext);
+
+export default function TodoProvider({ children }) {
+  const [todos, setTodo] = useState([]);
+
+  const addTodo = (todo) =>
+    setTodo([
+      ...todos,
+      {
+        id: v4(),
+        todo,
```

Comment:
I believe this should be named `description`. `todo` is the whole "object" containing infos about a TODO item! :)

Reply:
Either `description` or `title` are fine, you can chose it :)